### DISCUSSION TITLE
Button click should make visible a new container, but it is not the case with Cypress

### DISCUSSION THREAD

**Discussion by nudasilva** [cypress-io/cypress#25773] [Questions and Help]
Hello,

I block, when trying, via Cypress, to make visible a new container, by clicking a button.
This container (below the 'Send by email' button in the second screenshot, below, does not appear, with Cypress.
FYI: I can perform that, manually

![image](https://user-images.githubusercontent.com/97691378/218156448-cd3499f7-a7c6-4de6-9a68-6eb82d7cdf4b.png)

Click on this button =>

![image](https://user-images.githubusercontent.com/97691378/218156603-821aea2d-a9da-4ba4-a4b4-4d3bfabf8add.png)

I've tried with the following command:

**cy.get('.blue-button').click({ force: true });**

Could you help me, please?
Thanks in advance.

**Comment by nudasilva** [comment]
Finally, it is solved with a wait before this click, as execution is very fast via Cypress.

**Reply by jjhelguero** [reply]
Maybe due to resources, the click event was not attached to the element at the time of your cy.click()
